.github/workflows/perf.yml

```diff
@@ -13,12 +13,9 @@
 
 name: 'Performance test'
 on:
-  schedule:
```

Comment:
Do we no longer want to run a on a schedule?

Reply:
No. If it runs after each CI run as intended, it should be up-to-date and a scheduled run should not be needed.